### DISCUSSION TITLE
Can’t remove DRM from legit Kindle for Mac files

### DISCUSSION THREAD

**Discussion by PsiFi009** [apprenticeharper/DeDRM_tools#2160] [Q&A]
I have installed the latest app versions - Kindle for Mac 1.39.0; Calibre 6.7.1 including the plugins, DeDRM 7.2.1 and KFX Input 1.49.0 - running on a M1 Mac Studio with macOS 12.6.

I have tested this with a couple of AZW files downloaded from my Amazon cloud. They are readable in Kindle for Mac but I cannot remove the DRM in Calibre. I simply want to convert Kindle files into ePUB or DOCX format for reading on a different app on my iPad.

DRM removal is completely new for me. Many thanks in advance for any help you can give.

[Calibre .docx](https://github.com/apprenticeharper/DeDRM_tools/files/9796554/Calibre.docx)

**Comment by ZolaLa9** [comment]
1. This won't work with Kindle for Mac 1.39.0. The last version it works with is 1.31. 
2. You need to use [DeDRM 10.0.3](https://github.com/noDRM/DeDRM_tools/releases). (This GitHub by ApprenticeHarper is no longer being maintained. Go to [noDRM](https://github.com/noDRM/DeDRM_tools)'s GitHub for the latest plugin.)
3. More info here: https://www.mobileread.com/forums/showthread.php?t=283371
4. **_Important_ Mac-specific info here**: https://www.mobileread.com/forums/showpost.php?p=3819708&postcount=508

**Comment by PsiFi009** [comment]
Thanks so much for the comprehensive info. Most appreciated.
